src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -662,7 +691,7 @@ def _generate_properties(api_version, orchestrator_type, orchestrator_version, m
     return properties
 
 
-# pylint: disable=too-many-locals
+# pylint: disable=too-many-locals,too-many-statements
```

Comment:
you don't have to fix this now but the `too-many-statements` indicates that it is time to break up this method.

Reply:
Agreed. I added exactly one statement and that triggered the linter warning, so it's overdue for refactoring. I'll put it on my TODO list.